webpack.test.js

```diff
@@ -0,0 +1,20 @@
+var path = require('path')
```

Comment:
It looks like this webpack config is _nearly_ identical with our regular `webpack.config.js` file. Out of curiosity, does that compiled output not work?

Reply:
Good question!  Yes.  Some older browsers complaints about the arrow function and other ES6/7 syntaxes.  The original `webpack.config.js` was designed for publishing so some adjustments need to be made for testing.